frontend/src/metabase/containers/AddToDashSelectDashModal/AddToDashSelectDashModal.unit.spec.tsx

```diff
@@ -573,6 +659,107 @@ describe("AddToDashSelectDashModal", () => {
               }),
             ).toBeInTheDocument();
           });
+
+          describe('when "Create a new dashboard" option is clicked', () => {
+            beforeEach(async () => {
+              // "Create a new dashboard" option only renders when opening personal collections
+              userEvent.click(
+                screen.getByRole("heading", {
+                  name: PERSONAL_COLLECTION.name,
+                }),
+              );
+              userEvent.click(
+                screen.getByRole("heading", {
+                  name: "Create a new dashboard",
+                }),
+              );
+            });
+
+            it("should render only personal collections", async () => {
+              // Create a new dashboard modal
+              expect(
+                screen.getByRole("heading", { name: "New dashboard" }),
+              ).toBeInTheDocument();
+              userEvent.click(screen.getByTestId("select-button"));
+
+              const popover = screen.getByRole("tooltip");
+              expect(popover).toBeInTheDocument();
+              expect(
+                await within(popover).findByRole("heading", {
+                  name: PERSONAL_COLLECTION.name,
+                }),
+              ).toBeInTheDocument();
+              expect(
+                within(popover).queryByRole("heading", {
+                  name: "Our analytics",
+                }),
+              ).not.toBeInTheDocument();
+              expect(
+                within(popover).queryByRole("heading", {
+                  name: COLLECTION.name,
+                }),
+              ).not.toBeInTheDocument();
+            });
+
+            it('should not render "New collection" option', async () => {
```

Comment:
I suggest to be more specific with test name here because the next test (`when "New collection" option is clicked`) _seemingly_ contradicts this one.